binder/views.py

```diff
@@ -1365,6 +1376,18 @@ def binder_validation_error(self, obj, validation_error, pk=None):
 		})
 
 
+
+	def _abort_when_standalone_validation(self, request):
+		"""Raise a `BinderSkipSave` exception when this is a validation request."""
+		if 'validate' in request.GET and request.GET['validate'] == 'true':
+			if self.allow_standalone_validation:
+				params = QueryDict(request.body)
+				raise BinderSkipSave
+			else:
+				raise BinderRequestError('Standalone validation not enabled. You must enable this feature explicitly.')
```

Comment:
Can you add in the error a small description on how to enable this standalone_validation? e.g. add allow_standalone_validation = True on the model Foo

Reply:
See my latest commit